src/worker.c

```diff
@@ -102,7 +103,30 @@ header_cb(void *contents, size_t size, size_t nmemb, void *userp)
 	return realsize;
 }
 
-static int init(CURLM *cm, char *url, int64 id, HTAB *curlDataMap)
+static struct curl_slist *
+header_array_to_slist(ArrayType *array, struct curl_slist *headers)
+{
+	ArrayIterator iterator;
+	Datum value;
+	bool isnull;
+	char *header;
+
+	iterator = array_create_iterator(array, 0, NULL);
+
+	while (array_iterate(iterator, &value, &isnull))
+	{
+		if (isnull) continue;
+
+		header = TextDatumGetCString(value);
+		elog(DEBUG1, "Request header \"%s\"\n", header);
```

Comment:
probably don't want to inadvertently log auth info

Reply:
Makes sense. With the default log level (`WARNING`) this wouldn't be printed but this is redundant anyway when setting `CURLOPT_VERBOSE`.